synfig-core/src/modules/mod_svg/svg_parser.cpp

```diff
@@ -1229,6 +1234,34 @@ Svg_parser::parser_path_circle(const xmlpp::Element* nodeElement, const Style& s
 	return k;
 }
 
+std::list<BLine>
+Svg_parser::parser_line(const xmlpp::Element *nodeElement, const Style &style, const SVGMatrix &mtx)
+{
+	std::list<BLine> k;
+	if (!nodeElement)
+		return k;
+
+	double x1 = 0;
+	double x2 = 0;
+	double y1 = 0;
+	double y2 = 0;
+
+	try {
+		x1 = std::stod(nodeElement->get_attribute_value("x1"));
+		x2 = std::stod(nodeElement->get_attribute_value("x2"));
+		y1 = std::stod(nodeElement->get_attribute_value("y1"));
+		y2 = std::stod(nodeElement->get_attribute_value("y2"));
+	} catch (...) {
+		synfig::error("SVG Parser: Invalid <line> attribute: x1,y1,x2,y2 should be real values or percentages!");
```

Comment:
If the values can be percentages, will they be correctly interpreted?

Reply:
Currently almost anywhere.
I'll provide a PR for fix at everywhere I can locate XD